pages/start.tsx

```diff
@@ -107,243 +116,129 @@ const LoginStep1: React.FC<LoginStep1Props> = ({ handleLoginStep, user }) => {
-      handleLoginStep();
-    } catch (e) {
-      // If invalid, display error
-      console.error(e);
-    }
+  const handleUpdateInterest = async () => {
+    handleUpdateInterestsParent(selectedInterest);
+    handleLoginStep();
   };
 
   return (
     <>
-      <div className="login-step flex flex-col h-full gap-6">
+      <div className="login-step flex flex-col h-full lg:gap-28">
         <div>
           <div className="flex items-center gap-2 mb-4">
-            <Icon className="lg:hidden" IconImage={PATActiveIcon} size={48} />
+            <Icon className="lg:hidden" IconImage={ChooseInterestsActiveIcon} size={48} />
             <Title className="!text-sm !text-light-orange-9">Step Two</Title>
           </div>
           <div className="gap-2 mb-4">
-            <Title className="!text-2xl">Provide your Organization</Title>
+            <Title className="!text-2xl">Choose your interests</Title>
           </div>
           <div className="mb-4 text-left ">
             <Text className="!text-sm">
-              In order to provide fresh, and insightful data, we‘ll use your authenticated GitHub access token to fetch
-              public GitHub data. Here‘s how we‘re going to use your token:
+            Take a moment to select your interests to help us provide personalized project recommendations. By doing so, you&apos;ll find projects that match your skills and aspirations.
             </Text>
           </div>
-          <div className="flex flex-col gap-1">
-            <div className="flex gap-2 items-start">
-              <div className="mt-0.5 shrink-0">
-                <Icon IconImage={HighlightIcon} />
-              </div>
-              <div className="mt-0">
-                <Text className="!text-base !text-light-slate-12">Index insights from git data</Text>
-              </div>
-            </div>
-            <div className="flex gap-2 items-start">
-              <div className="mt-0.5 shrink-0">
-                <Icon IconImage={HighlightIcon} />
-              </div>
-              <div className="mt-0">
-                <Text className="!text-base !text-light-slate-12">
-                  Fetch basic GitHub information from Pull Requests and Issues
-                </Text>
-              </div>
-            </div>
-          </div>
-        </div>
-        <div className="flex flex-col gap-6">
-          <div>
-            <TextInput
-              placeholder="Organization Slug"
-              descriptionText="Grab the slug from GitHub, eg: 'open-sauced'."
-              classNames="!rounded-md !py-0.5 child:h-8 child:text-sm"
-              handleChange={handleSlugChange}
-            />
+          <div className="flex gap-3 flex-wrap">
+            {interestArray.map((topic, index) => (
+              <LanguagePill
+                onClick={() => handleSelectInterest(topic)}
+                classNames={`${(selectedInterest || []).includes(topic) && "bg-light-orange-10 text-white"}`}
```

Comment:
These are likely being overridden within the LanguagePill styles

```suggestion
                classNames={`${(selectedInterest || []).includes(topic) && "!bg-light-orange-10 !text-white"}`}
```